packages/main/src/components/FlexBox/FlexBox.jss.ts

```diff
@@ -2,6 +2,7 @@ export const styles = {
   flexBox: {
     display: 'flex'
   },
+  fitContainer: { width: '100%', height: '100%', backgroundColor: 'red' },
```

Comment:
I guess the background color is a leftover?

Reply:
Oops :sweat_smile:
It's removed now.